### PR TITLE
ddl: set low priority for adding the index operation

### PR DESCRIPTION
**Pull request by ciscoxll**

### CODE REVIEW

File: ddl/column.go [hunk 1]
```diff
@@ -304,7 +304,8 @@ func (d *ddl) addTableColumn(t table.Table, columnInfo *model.ColumnInfo, reorgI
 	for {
 		startTime := time.Now()
 		handles = handles[:0]
-		err = iterateSnapshotRows(d.store, t, version, seekHandle,
+		d.priority = 0
```
Comment (shenli) [inline ddl/column.go]: Please use the constants defined in kv/kv.go

File: domain/domain.go [hunk 2]
```diff
@@ -67,7 +68,7 @@ type Domain struct {
 // It returns the latest schema version, the changed table IDs, whether it's a full load and an error.
 func (do *Domain) loadInfoSchema(handle *infoschema.Handle, usedSchemaVersion int64, startTS uint64) (int64, []int64, bool, error) {
 	var fullLoad bool
-	snapshot, err := do.store.GetSnapshot(kv.NewVersion(startTS))
+	snapshot, err := do.store.GetSnapshot(kv.NewVersion(startTS), pb.CommandPri_Normal)
```
Comment (shenli) [inline domain/domain.go]: We should use high priority for load infoschema.

File: kv/fault_injection.go [hunk 3]
```diff
@@ -17,6 +17,7 @@ import (
 	"sync"
 
 	"golang.org/x/net/context"
+	pb "github.com/pingcap/kvproto/pkg/kvrpcpb"
```
Comment (shenli) [inline kv/fault_injection.go]: You can use the constants in kv.go.

**Comment by shenli** [comment]
Please fix the CI.

File: domain/domain.go (hunk 2 above)
Comment (ciscoxll) [reply domain/domain.go]: Change to high priority

File: ddl/column.go (hunk 1 above)
Comment (ciscoxll) [reply ddl/column.go]: Change with a constant

File: kv/fault_injection.go (hunk 3 above)
Comment (ciscoxll) [reply kv/fault_injection.go]: ok

File: kv/fault_injection.go [hunk 4]
```diff
@@ -74,8 +75,8 @@ func (s *InjectedStore) BeginWithStartTS(startTS uint64) (Transaction, error) {
 }
 
 // GetSnapshot creates an injected Snapshot.
-func (s *InjectedStore) GetSnapshot(ver Version) (Snapshot, error) {
-	snapshot, err := s.Storage.GetSnapshot(ver)
+func (s *InjectedStore) GetSnapshot(ver Version, priority pb.CommandPri) (Snapshot, error) {
+	snapshot, err := s.Storage.GetSnapshot(ver, pb.CommandPri_Normal)
```
Comment (winkyao) [inline kv/fault_injection.go]: use `kv.PriorityNormal`

File: ddl/ddl.go [hunk 5]
```diff
@@ -277,6 +278,7 @@ func newDDL(ctx context.Context, etcdCli *clientv3.Client, store kv.Storage,
 		ownerManager: manager,
 		schemaSyncer: syncer,
 		workerVars:   variable.NewSessionVars(),
+		priority:	  1,
```
Comment (zz-jason) [inline ddl/ddl.go]: Please use the constants defined in "kv/kv.go", too

File: ddl/index.go [hunk 6]
```diff
@@ -512,6 +514,7 @@ func newWorker(ctx sessionctx.Context, id, batch, colsLen, indexColsLen int) *wo
 		idxRecords:  make([]*indexRecord, 0, batch),
 		defaultVals: make([]types.Datum, colsLen),
 		rowMap:      make(map[int64]types.Datum, indexColsLen),
+		priority:    1,
```
Comment (zz-jason) [inline ddl/index.go]: ditto

**Comment by ciscoxll** [comment]
change parameters and fix the ci test problem 
The new branch is here #5972

**Pull request closed without merging**